compiler/test/compilable/traits.d

```diff
@@ -76,18 +76,26 @@ int foo();
 int foo(int);
 
 static assert(__traits(getLocation, __traits(getOverloads, traits, "foo")[1])[1] == 74);
+static assert(__traits(getLocation, __traits(getOverloads, foo)[1])[1] == 74);
+
+import imports.traits_a;
+import imports.traits_b;
+
+static assert(__traits(compiles, __traits(getOverloads, baz)[0]()));
+static assert(__traits(compiles, __traits(getOverloads, baz)[1](int.init)));
+static assert(__traits(compiles, __traits(getOverloads, baz)[2](string.init)));
```

Comment:
Shouldn't this be an error? Why is the first import chosen?

Reply:
It's not. baz is a tuple of three functions; two from the first import, one from the second.

Reply:
How does it get its order? Is it in order as they are added to the `OverloadSet` ?

Reply:
Yeah, order of iteration in `overloadApplyRecurse`.